File: test_dagger2/app/src/main/java/com/home/MainActivity.java
package com.home;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;

import com.m6world.test_dagger2.R;

import javax.inject.Inject;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: test_robolectric/app/src/main/java/com/m6world/test_robolectric/MainActivity.kt
package com.m6world.test_robolectric

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View
import android.widget.Button
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        button.setOnClickListener(View.OnClickListener {
            textView.text = "Robolectric Rocks!"
        })
    }

    fun getButton(): Button {
        return button;
    }

    fun getMessage(): String {
        return textView.text.toString();
    }
}
